medmorph_pha/api/fhir.py

```diff
@@ -14,22 +16,22 @@ def process_message():
     return response
 
 
-@blueprint.route('/', defaults={'relative_path': ''})
-@blueprint.route('/<path:relative_path>')
+@blueprint.route('/', defaults={'relative_path': ''}, methods=ALL_METHODS)
+@blueprint.route('/<path:relative_path>', methods=ALL_METHODS)
 def route_fhir(relative_path):
     backing_fhir_base_url = current_app.config['BACKING_FHIR_URL']
     backing_fhir_url = '/'.join((backing_fhir_base_url, relative_path))
     backing_headers = {}
     if 'Authorization' in request.headers:
         backing_headers = {'Authorization': request.headers['Authorization']}
 
-    # TODO pass all HTTP verbs, not just GET
-    backing_response = requests.get(
+    backing_response = remote_request(
+        method=request.method,
         url=backing_fhir_url,
         headers=backing_headers,
         params=request.args,
     )
-    return backing_response.json(), backing_response.status_code
+    return jsonify(backing_response.json())
```

Comment:
Does this still pass through the response code from HAPI? Without doing that, a `404` returned from HAPI will be be returned as a `200`, confusing to anyone querying the flask proxy

Reply:
>in general, as is, we don’t catch HAPI complaints, in fact, we’d probably miss them.

If we need to catch 5XX errors, maybe we could add some logging for those status codes